src/universal/components/Badge/Badge.js

```diff
@@ -31,11 +39,12 @@ Badge.defaultProps = {
   value: 0
 };
 
+const boxShadow = '1px 1px 2px rgba(0, 0, 0, .5)';
```

Comment:
should boxShadow live in `ui`? I feel like we'll wanna re-use the same 5 or so box shadows.